src/ucp/core/ucp_context.h

```diff
@@ -256,6 +256,12 @@ typedef struct ucp_context {
         /* Config environment prefix used to create the context */
         char                      *env_prefix;
 
+        struct {
+            /* Time period between keepalive rounds */
+            ucs_time_t            timeout;
+            /* Maximal number of endpoints to check on every round */
+            unsigned              num_eps;
+        } keepalive;
```

Comment:
why needed?

Reply:
used to operate by ucs_time_t values in timeouts

Reply:
i mean, why need num_eps?

Reply:
for consistency between config & config.ext
removed